run.py

```diff
@@ -0,0 +1,9 @@
+from fastapi import FastAPI
+
+from main.api.api import api_router
+from main.services.database.init_db import init_db
+
+init_db()
```

Comment:
SQLAlchemy will not attempt to create tables if these tables already exist in the bound engine.
I suggest moving this line from init_db.py `models.base_class.Base.metadata.create_all(bind=engine)` into session.py because having init_db() here is misleading.